pkgs/by-name/ni/nixos-rebuild-ng/README.md

```diff
@@ -35,40 +35,6 @@ an attempt of the rewrite.
   currently depends on `jq` for JSON parsing, while Python has `json` in
   standard library
 
-## Do's and Don'ts
-
-- Do: be as much of a drop-in replacement as possible
-- Do: fix obvious bugs
-- Do: improvements that are non-breaking
-- Don't: change logic in breaking ways even if this would be an improvement
```

Comment:
Do you intend to keep to these principles during the release cycle they co‐exist in? If so, I think we should leave this in. Otherwise LGTM as long as we’re prepared to revert if something major comes up.